content/acra/configuring-maintaining/key-storing/kv-stores.md

```diff
@@ -26,6 +26,11 @@ in terms that you will be able to manage all the keys from a single place.
 To mitigate the performance penalty of "doing network requests to get each key" you may use in-memory cache of keys,
 configured with `--keystore_cache_size` flag.
 
+{{< hint warning >}}
+**Note:**
+Currently, in-memory cache could be used only for keystore `v1`. 
```

Comment:
imho, would be great to mention what users will get if try to use. For example acra will not start.